### PR TITLE
chore(manifests): Removing privilege permissions from cluster-k8s-manifests

### PR DESCRIPTION
**Pull request by rajdas98**
## Proposed changes

Removing privilege permissions  from  cluster-k8s-manifests

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR that need to be merged before this (if any).

### CODE REVIEW

File: litmus-portal/cluster-k8s-manifest.yml [hunk 1]
```diff
@@ -1,10 +1,458 @@
...
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-view: "true"
+  name: argo-aggregate-to-view-for-litmusportal-server
+rules:
+- apiGroups: [argoproj.io]
+  resources: [workflows, workflows/finalizers, workflowtemplates, workflowtemplates/finalizers, cronworkflows, cronworkflows/finalizers, clusterworkflowtemplates, clusterworkflowtemplates/finalizers]
+  verbs: [get, list, watch]
```
Comment (ishangupta-ds) [inline litmus-portal/cluster-k8s-manifest.yml:36]: shouldn't this be at the agent level @rajdas98 ?

File: litmus-portal/cluster-k8s-manifest.yml [hunk 2]
```diff
@@ -1,10 +1,458 @@
...
+  verbs: [get, list, create, delete, update, watch]
+- apiGroups: [apps.openshift.io]
+  resources: [deploymentconfigs]
+  verbs: [get, list]
+- apiGroups: [apps]
+  resources: [deployments, daemonsets, replicasets, statefulsets]
+  verbs: [get, list]
+- apiGroups: [argoproj.io]
+  resources: [workflows, workflows/finalizers, workflowtemplates, workflowtemplates/finalizers, cronworkflows, cronworkflows/finalizers, clusterworkflowtemplates, clusterworkflowtemplates/finalizers, rollouts]
+  verbs: [get, list, create, delete, update, watch]
```
Comment (ishangupta-ds) [inline litmus-portal/cluster-k8s-manifest.yml:357]: this too can be at the agent level

File: litmus-portal/graphql-server/manifests/cluster/3a_agents_rbac.yaml [hunk 3]
```diff
@@ -130,21 +129,6 @@ subjects:
...
-roleRef:
-  apiGroup: rbac.authorization.k8s.io
-  kind: ClusterRole
-  name: litmus-admin
-subjects:
-  - kind: ServiceAccount
-    name: litmus-admin
-    namespace: #{AGENT-NAMESPACE}
----
 apiVersion: v1
```
Comment (ishangupta-ds) [inline litmus-portal/graphql-server/manifests/cluster/3a_agents_rbac.yaml:132]: we were using this for engines

File: litmus-portal/cluster-k8s-manifest.yml [hunk 4]
```diff
@@ -1,10 +1,458 @@
...
+rules:
+- apiGroups: [eventtracker.litmuschaos.io]
+  resources: [eventtrackerpolicies]
+  verbs: [create, delete, get, list, patch, update, watch]
+- apiGroups: [eventtracker.litmuschaos.io]
+  resources: [eventtrackerpolicies/status]
+  verbs: [get, patch, update]
+- apiGroups: ["", extensions, apps]
+  resources: [deployments, daemonsets, statefulsets, pods, configmaps]
+  verbs: [get, list, watch]
```
Comment (ishangupta-ds) [inline litmus-portal/cluster-k8s-manifest.yml:386]: this too can be at the agent level

File: litmus-portal/graphql-server/manifests/cluster/3a_agents_rbac.yaml (hunk 3 above)
Comment (rajdas98) [reply litmus-portal/graphql-server/manifests/cluster/3a_agents_rbac.yaml:132]: It's a duplicate spec in that files

File: litmus-portal/cluster-k8s-manifest.yml (hunk 2 above)
Comment (rajdas98) [reply litmus-portal/cluster-k8s-manifest.yml:357]: These permissions are required by the server to apply these permission.

File: litmus-portal/cluster-k8s-manifest.yml (hunk 4 above)
Comment (rajdas98) [reply litmus-portal/cluster-k8s-manifest.yml:386]: same-https://github.com/litmuschaos/litmus/pull/3017#discussion_r669711685

**Review by ishangupta-ds — APPROVED** [review]

File: litmus-portal/cluster-k8s-manifest.yml (hunk 1 above)
Comment (rajdas98) [reply litmus-portal/cluster-k8s-manifest.yml:36]: same- https://github.com/litmuschaos/litmus/pull/3017#discussion_r669711685

**Review by Jonsy13 — APPROVED** [review]

**Pull request merged**